src/components/popover/popover.tsx

```diff
@@ -174,52 +75,57 @@ export class Popover extends Component<PopoverProps, PopoverState> {
     document.removeEventListener('click', this._handleMouseEvent);
     document.removeEventListener('keydown', this._handleKeyboardEvent);
   };
-  private _getHoverEvents = () => {
-    if (this.props.triggerMode === PopoverTriggerMode.Hover) {
-      return {
-        targetEvents: {
-          onMouseEnter: this._handleMouseEnter,
-          onMouseLeave: this._handleMouseLeave,
-        },
-        popoverEvents: {
-          onMouseEnter: this._handleHoverOnPopover,
-          onMouseLeave: this._handleHoverOnPopover,
-        },
-      };
+
+  private _handleClickOnOption = (cb: () => void) => (event: OnClickEvent, byKeyboard?: boolean) => {
+    this._closePopover();
+    cb();
+    if (byKeyboard) {
```

Comment:
the condition doesn't uses
let's remove the condition or improve a11y (pass byKeyboard property to cb method, and if option selected by keyboard - focus textarea)